astroquery/sdss/tests/test_sdss_remote.py

```diff
@@ -11,6 +11,8 @@
 from ... import sdss
 from ...exceptions import TimeoutError
 
+dr_list = list(range(8, sdss.conf.default_release + 3))
```

Comment:
I think it would be better to hard wire the actual numbers here, as changing the default from DR14 doesn't necessarily mean that there are newer DRs, and vice versa, new releases may not mean that the default will change.

Reply:
Agreed.